src/Http/Requests/StoreConsultationRequest.php

```diff
@@ -6,26 +6,27 @@
 use EscolaLms\Consultations\Enum\ConsultationStatusEnum;
 use EscolaLms\Consultations\Models\Consultation;
 use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Facades\Gate;
 use Illuminate\Validation\Rule;
 
 class StoreConsultationRequest extends FormRequest
 {
     public function authorize(): bool
     {
-        $user = auth()->user();
-        return isset($user) ? $user->can(ConsultationsPermissionsEnum::CONSULTATION_CREATE, Consultation::class) : true;
+        return Gate::allows(ConsultationsPermissionsEnum::CONSULTATION_CREATE, Consultation::class);
     }
 
     public function rules(): array
     {
         return [
             'base_price' => 'integer',
             'name' => 'required|string|max:255|min:3',
```

Comment:
For consistency, either use `=> [rule1, rule2]` or `=> 'rule1|rule2'` everywhere and not mix these two styles... and array is probably better because string is being exploded into array during validation anyway.